include/sway/input/text_input.h

```diff
@@ -59,6 +61,7 @@ void sway_input_method_relay_init(struct sway_seat *seat,
 	struct sway_input_method_relay *relay);
 
 void sway_input_method_relay_finish(struct sway_input_method_relay *relay);
+void sway_input_method_relay_finish_input_method(struct sway_input_method_relay *relay);
```

Comment:
Could we keep this `static` inside `text_input.c`, since it's not called elsewhere?